MorphicServer/Preferences.cs

```diff
@@ -33,15 +32,15 @@ namespace MorphicServer
     public class Preferences : Record
     {
         [JsonPropertyName("user_id")]
-        public string? UserId { get; set; }
+        public string UserId { get; set; } = null!;
```

Comment:
Do we need to make this non null for any reason?  It's not at risk of getting wiped out.  I was defaulting to a model design of everything is nullable unless there's a good reason not to.  It seems like we might end up storing preferences for non-user entities like groups or orgs down the road and want multiple nullable fields for back links, which are mainly there for convenience.

Reply:
No reason, I just figured it was required. I'll revert and add a comment.